hook.py

```diff
@@ -19,72 +22,177 @@
 EXECUTORS = dict(command_prompt='cmd', sh='sh', powershell='psh', bash='sh')
 RE_VARIABLE = re.compile('(#{(.*?)})', re.DOTALL)
 
+DATA_DIR = 'plugins/atomic/data/'
+
 
 async def enable(services):
-    data_svc = services.get('data_svc')
+    # we only ingest data once, and save new abilities in the data/ folder of the plugin
+    if os.path.exists(DATA_DIR) and os.listdir(DATA_DIR):
+        return
+
     repo_dir = 'plugins/atomic/atomic-red-team/'
     if not os.path.exists(repo_dir) or not os.listdir(repo_dir):
         repo_url = 'https://github.com/redcanaryco/atomic-red-team.git'
         check_call(['git', 'clone', '--depth', '1', repo_url, repo_dir], stdout=DEVNULL, stderr=STDOUT)
 
```

Comment:
kick this into the `atomic_svc.clone_atomic_red_team_repo()` function

Reply:
fixed in f6ce965